tests/engineFactory.py

```diff
@@ -1,140 +0,0 @@
-    """
-    Initializes and returns SQLAlchemy engine using values provided through the
-    file. The file must contain "database" section with key "url" defined.
-    Optionally it can contain **kwargs that can be passed to the SQLAlchemy
-    create_engine(). For  details see
-    http://docs.sqlalchemy.org/en/rel_1_0/core/engines.html
-
-    Optional parameters drivername, username, password, host, port, database,
-    query can be used to overwrite values from the file. A typical usecase:
-    one can use drivername, username, password, host, port from the file,
-    and pass database name.
-
-    Note, if mysql sees "localhost" it switches to using socket, even if port is
-    specified. Commonly used way around it is to specify "127.0.0.1" as port for
-    local access.
-
-    Example file:
-
-    [database]
-    url = mysql+mysqldb://joe:myPassword@localhost:3306/?unix_socket=/tmp/mysql.sock
-    echo = yes
-    pool_size = 5
-
-    Raises IOError if the file does not exists.
-    Raises ConfigParser exceptions (such as NoSectionError)
-    """
-    fileName = os.path.expanduser(fileName)
-    parser = ConfigParser()
-    with open(fileName) as config:
-        # readfp is deprecated in Python3.x, read_file is not in Python2
-        read_file = getattr(parser, "read_file", None) or getattr(parser,
-                                                                  "readfp")
-        read_file(config, fileName)
-    try:
-        options = dict(parser.items("database"))
-    except NoSectionError:
-        log.error("File %s does not contain section 'database'", fileName)
-        raise
-
-    if drivername or username or password or host or port or database or query:
-        url = make_url(options['url'])
-        if drivername:
-            url.drivername = drivername
-        if username:
-            url.username = username
-        if password:
-            url.password = password
-        if host:
-            url.host = host
-        if port:
-            url.port = port
-        if database:
-            url.database = database
-        if query:
-            url.query = query
-        options['url'] = url
-
-    return sqlalchemy.engine_from_config(options, "")
-
-################################################################################
-
-
-def getEngineFromArgs(drivername="mysql+mysqldb",
-                      username=None,
-                      password=None,
-                      host=None,
-                      port=None,
-                      database=None,
-                      query=None,
-                      **engineKVArgs):
-    """
-    Initializes and returns SQLAlchemy engine using provided values.
-
-    To specify socket, use query={"unix_socket": "/the/socket.file"}
-
-    Note, if mysql sees "localhost" it switches to using socket, even if port is
-    specified. Commonly used way around it is to specify "127.0.0.1" as port for
-    local access.
-    """
-    url = URL(drivername=drivername,
```

Comment:
What are some of these files that are deleted?  Is this some other unrelated cleanup?

Reply:
Are you deleting all the tests because now we're using sodalint?  Does that cover all of this?

Reply:
These are the dependencies of old unit tests that use sqlachemy and mysql connect to metaserv to unit test FITS files and its metadata (headers).  As @brianv0 pointed out, that's some really OLD proprietary tests, that are no longer relevant in our new VO way of doing things.  The metadata tests, for instance, will be replaced by unit tests with sia.

Reply:
Deleting these sounds like it should be a distinct commit with an explanation as to why they are being deleted in the commit message.

Reply:
agree.